drivers/npmx/npmx_shell.c

```diff
@@ -1559,6 +1559,36 @@ static int cmd_vbusin_status_connected_get(const struct shell *shell, size_t arg
 	return 0;
 }
 
+static int cmd_vbusin_status_cc_get(const struct shell *shell, size_t argc, char **argv)
+{
+	ARG_UNUSED(argc);
+	ARG_UNUSED(argv);
+
+	npmx_instance_t *npmx_instance = npmx_driver_instance_get(pmic_dev);
+
+	if (npmx_instance == NULL) {
+		shell_error(shell, "Error: shell is not initialized.");
+		return 0;
+	}
+
+	npmx_vbusin_cc_t cc1;
+	npmx_vbusin_cc_t cc2;
+	npmx_vbusin_t *vbusin_instance = npmx_vbusin_get(npmx_instance, 0);
+	npmx_error_t err_code = npmx_vbusin_cc_status_get(vbusin_instance, &cc1, &cc2);
+
+	if (check_error_code(shell, err_code)) {
+		if (cc1 == NPMX_VBUSIN_CC_NOT_CONNECTED) {
+			shell_print(shell, "Value: %d.", cc2);
```

Comment:
String message is better here: 
`shell_print(shell, "Value: %s.", npmx_vbusin_cc_status_map_to_string(cc2));`

Reply:
Yes, I agree, but I am not sure if people working on PowerUp would not prefer the returned value to be an integer. @masz-nordic what do you think?

Reply:
Integer is preferred.